metrics/circonus.go

```diff
@@ -0,0 +1,132 @@
+package metrics
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"sync"
+	"time"
+
+	cgm "github.com/circonus-labs/circonus-gometrics"
+)
+
+type cgmProvider struct {
+	metrics *cgm.CirconusMetrics
+	prefix  string
+}
+
+type cgmTimer struct {
+	metrics *cgm.CirconusMetrics
+	name    string
+}
+
+var (
+	circonus *cgmProvider
+	once     sync.Once
+)
+
+const serviceName = "fabio"
```

Comment:
Isn't this the value from `metrics.circonus.apiapp`? If not, then either make it configurable or inline it below instead of using `fmt.Sprintf`